Reader

```diff
@@ -0,0 +1,46 @@
+import UIKit
+
+protocol ReaderDelegate: class {
+    func didReadData(data: Data)
+}
+
+class Reader{
+    var file: String!
```

Comment:
`file` can be declared `let` and please avoid implicitly unwrapped when possible, here you can declare file like `let file: URL`